pkg/controller/volume/attachdetach/util/util.go

```diff
@@ -236,7 +236,7 @@ func ProcessPodVolumes(pod *v1.Pod, addVolumes bool, desiredStateOfWorld cache.D
 	for _, podVolume := range pod.Spec.Volumes {
 		volumeSpec, err := CreateVolumeSpec(podVolume, pod, nodeName, volumePluginMgr, pvcLister, pvLister, csiMigratedPluginManager, csiTranslator)
 		if err != nil {
-			klog.V(10).Infof(
+			klog.Errorf(
```

Comment:
Can be modified to klog.ErrorS

Reply:
This used to be log level 10 but pushing it to Errorf will make it a lot more frequent. I think this must have been a common/normal error for it to have been at 10.